Tribler/Core/Modules/MetadataStore/serialization.py

```diff
@@ -195,17 +198,112 @@ def from_unpack_list(cls, metadata_type, reserved_flags, public_key,
 
     def to_dict(self):
         dct = super(ChannelNodePayload, self).to_dict()
-        dct.update({
-            "id_": self.id_,
-            "origin_id": self.origin_id,
-            "timestamp": self.timestamp
-        })
+        dct.update(
+            {"id_": self.id_,
+             "origin_id": self.origin_id,
+             "timestamp": self.timestamp
+             })
         return dct
 
+
+class MetadataNodePayload(ChannelNodePayload):
```

Comment:
Reasonable

Reply:
Hmm... At first, I thought you said that about the ORM classes. I'll provide comment on these.
However, do you really think providing comments that just copy-paste the names of the classes make the code more readable, or more maintainable? Really?

```python
class SignedPayload(Payload):
    """
    Payload for metadata.
    """

    format_list = ['H', 'H', '64s']

class ChannelNodePayload(SignedPayolad):
    """
    Payload for ChannelNode class entries.
    """

    format_list = ['H', 'H', '64s']
```

Really? 

Just say "Yes", and I'll gladly add them :wink:.

Reply:
Ok, for payloads it's not really necessary I would say but I noticed that there was a comment for another (payload) class, so that's why I asked.

Reply:
It was you, who left that comment originally...

Reply:
I wasn't referring to the Github comment. I saw that there was another docstring so I wondered why there was no docstring under the class for which I made the initial Github comment.

Reply:
I removed you original comments for all the payloads but first, because I find those not to add any new information for the reader, but instead obscuring the code structure. For me, it looks like this:
![sign](https://user-images.githubusercontent.com/2509103/63117666-8ba28c80-bf9c-11e9-8137-62693b229800.jpg)